src/test/java/org/folio/innreach/controller/ContributionCriteriaControllerTest.java

```diff
@@ -42,6 +42,17 @@ void return200HttpCode_on_createCriteriaConfiguration() {
     removeContributionCriteriaConfiguration();
   }
 
+  @Test
+  void return200HttpCode_on_createCriteriaConfigurationWithoutExcludedLocations() {
```

Comment:
probably test name return20**1**...() would suit better, since the test expects 201 status code -  `HttpStatus.CREATED` in the assert below